index.js

```diff
@@ -6,11 +6,13 @@ See the accompanying LICENSE file for terms.
 
 'use strict';
 
-var isRegExp = require('util').isRegExp;
+var isRegExp    = require('util').isRegExp;
+var randomBytes = require('randombytes');
```

Comment:
My intention is node-only. But it's true that I don't know if people are bundling it for the browser. So let's stick with what you have using `randombytes`.